pkg/cvo/upgradeable.go

```diff
@@ -410,6 +409,24 @@ func (check *clusterAdminAcksCompletedUpgradeable) Check() *configv1.ClusterOper
 	return nil
 }
 
+// getCurrentVersion determines and returns the cluster's current version by iterating through the
+// provided update history until it finds the first version with update State of Completed. If a
+// Completed version is not found the version of the oldest history entry, which is the originally
+// installed version, is returned. If history is empty the empty string is returned.
+func getCurrentVersion(history []configv1.UpdateHistory) string {
```

Comment:
Why shift this out of `pkg/payload/precondition/clusterversion/upgradeable.go` and into `pkg/cvo`?  Not a big deal either way, but `pkg/cvo` has so much going on already, that I'm in favor of pushing as much as we can out into helper packages.